packages/number-input/src/ErrorIcon/ErrorIcon.styles.ts

```diff
@@ -0,0 +1,76 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { transitionDuration } from '@leafygreen-ui/tokens';
+
+import { wrapperClassName } from '../Input/Input.styles';
+import { Size } from '../NumberInput/NumberInput.types';
+
+export const sizeStyles: Record<Size, string> = {
+  [Size.XSmall]: css`
+    translate: 4px 0;
+  `,
+  [Size.Small]: css`
+    translate: 4px 0;
+  `,
+  [Size.Default]: css`
+    translate: 4px 0;
+  `,
+};
```

Comment:
Looks like this can be all a part of a `baseStyles` that's not dependent on `size`

Reply:
removing